src/shared/sagas/auth.js

```diff
@@ -22,7 +22,7 @@ function* authenticate({ username, password, provider }) {
 
     yield put(signInComplete({ attributes: response, provider }));
   } catch (error) {
-    yield put(signOutError({ provider, error }));
+    yield put(signOutError({ provider, error: error.message }));
```

Comment:
with the fix you've done above, there is no need to do this. also, that would be better to pass the full `error` object.